learn/what_is_meilisearch/features.md

```diff
@@ -18,51 +18,48 @@ Meilisearch's default [relevancy rules](/learn/core_concepts/relevancy.md) are d
-Read more about synonyms in [this dedicated guide](/learn/configuration/synonyms.md).
-
-## Highlighting
+## Geosearch
 
-[Highlight](/reference/api/search.md#attributes-to-highlight) query terms so that matches pop out to the eye. Users don't need to read the entire text to find the match.
+Explore the physical world. [Geosearch](/learn/advanced/geosearch.md) allows you to filter and sort results based on their geographic location.
 
-## Filters
+## Filtering
 
-Meilisearch allows you to define [filters](/learn/advanced/filtering_and_faceted_search.md) so you can filter through the results based on user-defined criteria.
+Create [filters](/learn/advanced/filtering_and_faceted_search.md) to refine results based on user-defined criteria.
 
 ## Faceting
 
-[Faceted search](/learn/advanced/filtering_and_faceted_search.md) allows you to classify search results into categories and to build intuitive navigation interfaces.
+Build intuitive navigation interfaces with [faceted search](/learn/advanced/filtering_and_faceted_search.md).
 
 ## Sorting
 
-[Sort search results](/learn/advanced/sorting.md) at query time and let users choose which types of results they want to see first.
-
-## Placeholder search
+[Sort search results](/learn/advanced/sorting.md) at query time and let users choose which results they want to see first.
 
-If you make a search without inputting any query words, Meilisearch will return all the documents in that index sorted by its [custom ranking rules](/learn/core_concepts/relevancy.md#custom-rules) and [sorting rules](/learn/advanced/sorting.md#sorting). This feature is called **placeholder search**.
+## API key management
 
-Placeholder searches are particularly effective when used with other features such as [faceting or filtering](/learn/advanced/filtering_and_faceted_search.md#filters-or-facets), which allow users to narrow their searches and browse by category. You can read more about this feature in our article on [search parameters](/reference/api/search.md#placeholder-search).
+[Protect your instance with API keys](/learn/security/master_api_keys.md). Control which users can access what indexes, routes, and endpoints.
 
-## Phrase search
+## Multitenancy
 
-If you enclose search terms in double quotes (`"`), Meilisearch will only return documents that contain those terms in the order they were given. This gives users the option to make more precise search queries.
+Manage complex multi-user applications handling sensitive data. [Tenant tokens](/learn/security/tenant_tokens.md) help you decide which documents each one of your users can search.
 
-Phrase search is particularly useful when looking for a single result, such as searching for a book by ISBN. It is also possible to combine phrase searches with Meilisearch's basic syntax so only parts of a query are matched strictly.
+## Synonyms
 
-## API key management
+[Define synonyms](/learn/configuration/synonyms.md) and craft a tailored, intuitive search experience.
```

Comment:
I personally think it makes sense to put synonyms much higher. Granted it's not a stunning feature, but it's a short one and topically much more connected with features like typo tolerance than it is w/ API keys etc.

Reply:
Thing is, the higher we place these, the lower we place security features. I feel both synonyms and highlighting are features in the way being able to type full sentences is a text editor feature.

Anyway, I think we need a better way of informing the order of features in this page. Perhaps we can order according to the number of page views their dedicated guides/reference pages get?

Reply:
Have you considered having an extra title to arrange the features in sections?

e.g `## Security`

 It might help to arrange it and make it easier for someone coming in for the first time to read.

Reply:
I agree with @gmourier , arranging the features in sections sounds like a good idea! However, we do want to get this PR merged today, so it might be better to do that in a separate PR.

@guimachiavelli For now, could you please move synonyms and highlighting to their previous location following typo tolerance? Since we're not too sure about which features we want to highlight or the order in which we want to present them, it makes sense to group features by topic for now. Thank you 🙏🏻